### ISSUE TITLE
Feature request: Animation

### ISSUE THREAD

**Issue by pannet1** [BoboTiG/py-candlestick-chart#19]
Is it possible to run a continuous streaming candle with python

**Comment by BoboTiG** [comment]
Of course, check the Rich example: https://github.com/BoboTiG/py-candlestick-chart/blob/main/examples/integrate-with-rich.py

**Comment by pannet1** [comment]
thanks. i will try. may be asking for too much. is there is possibility to add technical indicators from [TA-Lib](https://ta-lib.org/functions/) or something similar.
